packages/lambda-xyz/src/__test__/wmts.capability.test.ts

```diff
@@ -1,142 +1,120 @@
+            );
+            o(x.find('TileWidth')?.toString()).equals(`<TileWidth>256</TileWidth>`);
+            o(x.find('TileHeight')?.toString()).equals(`<TileHeight>256</TileHeight>`);
+            o(x.find('MatrixWidth')?.toString()).equals(`<MatrixWidth>${tileCount}</MatrixWidth>`);
+            o(x.find('MatrixHeight')?.toString()).equals(`<MatrixHeight>${tileCount}</MatrixHeight>`);
+        }
 
-        o(tileMatrices[10].toString()).equals(
-            '<TileMatrix>\n' +
-                '  <ows:Identifier>10</ows:Identifier>\n' +
-                '  <ScaleDenominator>545978.7734658204</ScaleDenominator>\n' +
-                '  <TopLeftCorner>-20037508.342789244 20037508.342789244</TopLeftCorner>\n' +
-                '  <TileWidth>256</TileWidth>\n' +
-                '  <TileHeight>256</TileHeight>\n' +
-                '  <MatrixWidth>1024</MatrixWidth>\n' +
-                '  <MatrixHeight>1024</MatrixHeight>\n' +
-                '</TileMatrix>',
-        );
+        compareMatrix(tileMatrices[0], '0', 1, 559082264.028717);
+        compareMatrix(tileMatrices[10], '10', 1024, 545978.773465544);
 
-        const xml = buildWmtsCapability('https://basemaps.test', req, Provider, TileSets.get('aerial@beta_3857')!)!;
+        const xml = WmtsCapabilities.toXml('https://basemaps.test', Provider, [tileSet], apiKey) ?? '';
 
-        o(xml).deepEquals('<?xml version="1.0"?>\n' + raw.toString());
+        o(xml).deepEquals('<?xml version="1.0"?>\n' + raw?.toString());
 
         o(createHash('sha256').update(Buffer.from(xml)).digest('base64')).equals(
-            'Y6X5Q9VqbY+Y7wRUhKMCIk/V7OQIkCXD20xGmsiuFlc=',
+            'NqBsiXcdQ6LtRCcvqS7pizuUk56N6crp3bWpmlh5iZs=',
         );
     });
 
     o('should return null if not found', () => {
-        const req = mockRequest('/v1/tiles/aerial/4326/WMTSCapabilities.xml');
-        const ts = new TileSet('aerial', Epsg.Nztm2000);
-        addTitleAndDesc(ts);
-
-        o(buildWmtsCapability('basemaps.test', req, Provider, ts)).equals(null);
+        const ts = new FakeTileSet('aerial', { code: 9999 } as Epsg);
+        o(() => WmtsCapabilities.toXml('basemaps.test', Provider, [ts])).throws('Invalid projection: 9999');
     });
 
     o('should allow individual imagery sets', () => {
-        const raw = buildWmtsCapabilityToVNode(
-            'https://basemaps.test',
-            mockRequest('/v1/tiles/01E7PJFR9AMQFJ05X9G7FQ3XMW/3857/WMTSCapabilities.xml'),
-            Provider,
-            TileSets.get('01E7PJFR9AMQFJ05X9G7FQ3XMW_3857')!,
-        )!;
+        const raw = new WmtsCapabilities('https://basemaps.test', Provider, [tileSetImagery]).toVNode();
 
-        const tms = raw?.find('TileMatrixSet', 'ows:Identifier')!;
+        const tms = raw?.find('TileMatrixSet', 'ows:Identifier');
 
-        o(tms.textContent).equals('GoogleMapsCompatible');
+        o(tms?.textContent).equals('WebMercatorQuad');
 
```

Comment:
Why the name change

Reply:
I used http://docs.opengeospatial.org/is/17-083r2/17-083r2.html#73 for the source of our tile matrix definition. it uses `WebMercatorQuad`